lib/src/excel.dart

```diff
@@ -1,36 +1,128 @@
 @JS()
 library excel;
 
+import 'package:fixnum/fixnum.dart';
 import 'package:js/js.dart';
 
 import 'office_js.dart';
+import 'proto/insertion_order.pb.dart';
 
 /// Service class that provides dart functions to interact with Excel.
 class ExcelDart {
-  ExcelDart._private();
+  static final _startAddressInt = 'A'.codeUnitAt(0);
+  static final _fontName = 'Roboto';
+  static final _fontSize = 12;
+  static final _horizontalAlignment = 'Center';
+  static final _borderStyle = 'Continuous';
 
-  static final ExcelDart _singleton = ExcelDart._private();
+  static final _tableHeader = [
```

Comment:
does this have to keep in line with the row proto? what happens if they mismatch?

Reply:
tableHeader will need to be in line with the _generateTableRow() function. Changing the protobuff definitions can affect the _generateTableRow() function (invalid getters etc). I will add a comment specifying this.